packages/neo-one-cli/src/InteractiveCLI.ts

```diff
@@ -275,6 +279,14 @@ export class InteractiveCLI {
                 },
               });
 
+              if (verSpinner !== undefined) {
+                if (newerVersion === undefined) {
+                  verSpinner.succeed(`@neo-one/server is current`);
+                } else {
+                  verSpinner.fail(`A newer version (${newerVersion}) of @neo-one/server is available for download`);
```

Comment:
Is there a warning instead? It's not really a failure for it to be out of date. Also would be good to link to documentation that explains how to update.